sw/device/tests/sim_dv/sleep_pin_wake_test.c

```diff
@@ -78,8 +75,12 @@ bool test_main(void) {
   CHECK_DIF_OK(dif_pinmux_init(
       mmio_region_from_addr(TOP_EARLGREY_PINMUX_AON_BASE_ADDR), &pinmux));
 
+  // Randomly pick one of the wakeup detectors.
```

Comment:
just a curious question, is the idea here that you randomly select which actual detector to use in the C test, instead of having that information be created in the vseq and passed over? I guess the vseq side has no reason to care which detector we use, as long as the wakeup pin selected actually wakes up right?

Reply:
Right - the SV side doesn't need to randomize of be aware of the chosen wakeup detector since that randomization can be done in C itself. The SV side only needs to know which pad to wiggle to wake up the chip.

Reply:
makes sense to me! thanks for explaining.